types/bun/README.md

```diff
@@ -0,0 +1,39 @@
+# TypeScript types for Bun
+
+<p align="center">
+  <a href="https://bun.sh"><img src="https://bun.sh/logo@2x.png" alt="Logo"></a>
+</p>
+
+These are the type definitions for Bun's JavaScript runtime APIs.
+
+# Installation
+
+Install the `@types/bun` npm package:
+
+```bash
+# yarn/npm/pnpm work too, "@types/bun" is an ordinary npm package
+bun add -d @types/bun
+```
+
+# Usage
+
+Add this to your `tsconfig.json` or `jsconfig.json`:
+
+```jsonc-diff
+  {
+    "compilerOptions": {
++     "types": ["@types/bun"]
```

Comment:
This is more canonically written `"bun"`.

Reply:
But, I thought half of the point was to not have to manually specify anything?

Reply:
yeah, i think this entire "usage" section should be removed